src/components/DiffSection.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import CoffeeShopCommercial from "../assets/CoffeeShopCommercial.mp4";
+
+function DiffSection(){
+    return (
+        <div className="DiffVid">
+            <video src={CoffeeShopCommercial} autoPlay  loop>
+            
+            <track kind="captions"/>
+            </video>
+            <div className="WelcomeMessage">
+                Welcome to our Dilmun
+            </div>
```

Comment:
This should be "Welcome to Dilmun" but for some reason, it is not shown in anywhere. We ideally need to have some writing on the video something like "Check the video that is created by one of our satisfied clients which summarizes how a typical day passes in one of our bricks-and-mortars.". And there should also be a button below it that will open up the video in another tab. Please also change the video to something more shipping/trade related one. Also there should be no sound when it plays, and video height must be restricted to something like 400 px, its width to full screen view on large and very large screens, on medium and small screens, it should be an image instead. 

The component is, as i mentioned above, not responsive. Please make it responsive using tailwind.